package.json

```diff
@@ -1,11 +1,8 @@
 {
   "name": "tldr-lint",
   "version": "0.0.7",
-  "description": "Linter for TLDR pages.",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/rubenvereecken/tldr-lint.git"
-  },
+  "description": "A linting tool to validate tldr pages",
+  "repository": "rubenvereecken/tldr-lint",
```

Comment:
Shouldn't we have a full URL here?

Reply:
This is the [official shorter way](https://docs.npmjs.com/files/package.json#repository) of writing it.  But now I see even the repository URL is wrong. :smile:

Reply:
Cons:

 - If the preference towards a specific domain name changes in the future, this will break
 - It isn't obvious as to the exact location of the repository
 - If we include a full URL here, an interested user can click / copy+paste the link directly

Reply:
The preference hasn't changed ever since it has been implemented, and is unlikely to change given the number of users of the feature.
Since no one actually reads `package.json`, but the data parsed by npm, both points are pretty weak. This is an example output:
```
$ npm view percent repository
{ type: 'git', url: 'git+https://github.com/zdroid/percent.git' }
```
(same as the full syntax)